package.json

```diff
@@ -80,6 +85,7 @@
     }
   },
   "dependencies": {
+    "@angular/forms": "^5.2.9",
```

Comment:
We would want this to be a peer dependency.

Reply:
Done.